testdata/scripts/ignore-implicit-vendor.txt

```diff
@@ -0,0 +1,59 @@
+gofumpt -d formatted.go.golden
+! stdout .
+
+mkdir vendor
+
+# format explicit vendor dir
+cp orig.go.golden vendor/foo.go
+gofumpt -w vendor
+cmp vendor/foo.go formatted.go.golden
```

Comment:
probably easiest to use `-l` and then check `stdout 'vendor/foo.go'`

Reply:
another advantage of that approach is you don't modify files in place, so you avoid extra `cp` lines

Reply:
thanks! PTAL